cli-commands/vm/post/show.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+UbiRodish.on("vm").run_on("show") do
+  fields = %w[id name state location size unix_user storage_size_gib ip6 ip4_enabled ip4 private_ipv4 private_ipv6 subnet firewalls].freeze.each(&:freeze)
+  firewall_fields = %w[id name description location path firewall_rules].freeze.each(&:freeze)
+  firewall_rule_fields = %w[id cidr port_range].freeze.each(&:freeze)
+
+  options("ubi vm location/(vm-name|_vm-ubid) show [options]", key: :vm_show) do
+    on("-f", "--fields=fields", "show specific fields (default: #{fields.join(",")})")
+    on("-r", "--firewall_rule_fields=fields", "show specific fields (default: #{firewall_rule_fields.join(",")})")
+    on("-w", "--firewall_fields=fields", "show specific fields (default: #{firewall_fields.join(",")})")
```

Comment:
What do you think about combining them into `fields` and get firewall fields with a prefix. i.e. `-f id,name,firewalls:name`

Reply:
How would you handle multiple nested fields in that case? `-f id,name,firewalls:name,firewalls:id`?  There aren't many nested fields, so I guess the repetitiveness issue isn't too severe.  However, it adds complexity.  The only reason I can think to do it is if you wanted to use `-r` or `-w` for other things.